specs/eventing/control-plane.md

```diff
@@ -0,0 +1,1097 @@
+   `deadLetterSink` in the delivery options. If no `deadLetterSink` is
+   specified, the event is dropped.
+
+   The implementation MAY set additional attributes on the event or wrap the
+   failed event in a "failed delivery" event; this behavior is not (currently)
+   standardized.
+
+   If delivery of the reply event fails with a retriable error, the delivery to
+   the `deadLetterSink` SHOULD be retried up to `retry` times, following the
+   `backoffPolicy` and `backoffDelay` parameters if specified. Alternatively,
+   implementations MAY use an equivalent internal mechanism for delivery (for
+   example, if the `ref` form of `deadLetterSink` points to a compatible
+   implementation).
+
+# Detailed Resources
+
+## Broker v1
+
+### Metadata:
+
+Standard Kubernetes
+[metav1.ObjectMeta](https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.21/#objectmeta-v1-meta)
+resource.
+
+### Spec:
+
+<table>
+  <tr>
+    <td><strong>Field Name</strong></td>
+    <td><strong>Field Type</strong></td>
+    <td><strong>Description</strong></td>
+    <td><strong>Schema Requirement</strong></td>
+  </tr>
+  <tr>
+    <td><code>config</code></td>
+    <td>KReference<br/>(Optional)</td>
+    <td>A reference to an object which describes the configuration options for the Broker (for example, a ConfigMap).</td>
+    <td>RECOMMENDED</td>
+  </tr>
+  <tr>
+    <td><code>delivery</code></td>
+    <td>DeliverySpec<br/>(Optional)</td>
+    <td>A default delivery options for Triggers which do not specify more-specific options.</td>
+    <td>REQUIRED</td>
+  </tr>
+</table>
+
+### Status
+
+<table>
+  <tr>
+    <td><strong>Field Name</strong></td>
+    <td><strong>Field Type</strong></td>
+    <td><strong>Description</strong></td>
+    <td><strong>Schema Requirement</strong></td>
+  </tr>
+  <tr>
+    <td><code>conditions</code></td>
+    <td><a href="#error-signalling">See Error Signalling</a></td>
+    <td>Used for signalling errors, see link. Conditions of types Ready MUST be present.</td>
+    <td>REQUIRED</td>
+  </tr>
+  <tr>
+    <td><code>observedGeneration</code></td>
+    <td>int64</td>
+    <td>The latest metadata.generation that the reconciler has attempted. If <code>observedGeneration</code> is updated, <code>conditions</code> MUST be updated with current status in the same transaction.</td>
+    <td>REQUIRED</td>
+  </tr>
+  <tr>
+    <td><code>address</code></td>
+    <td>duckv1.Addressable</td>
+    <td>Address used to deliver events to the Broker.</td>
+    <td>REQUIRED</td>
+  </tr>
+  <tr>
+    <td><code>deadLetterSinkUri</td>
+    <td>URL (string)</td>
+    <td>If <code>spec.delivery.deadLetterSink</code> is specified, the resolved URL of the dead letter address.</td>
+    <td>REQUIRED</td>
+  </tr>
+</table>
+
+## Trigger v1
+
+### Metadata:
+
+Standard Kubernetes
+[metav1.ObjectMeta](https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.21/#objectmeta-v1-meta)
+resource.
+
+### Spec:
+
+<table>
```

Comment:
why not markdown tables?

Reply:
Good question -- serving used tables here, and the descriptions sometimes get a bit long for Markdown.

I'd be willing to try converting if it seems like it would be a win, though.